.github/workflows/ci.yml

```diff
@@ -85,3 +85,41 @@ jobs:
         with:
           severity: error
           ignore_paths: third-party
+  dokken-recipes:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - alinux2
+          - centos7
+          #- centos8
```

Comment:
Let's try to avoid dead comments in the code.

Reply:
yep it was just a test, thanks, addressed in the https://github.com/aws/aws-parallelcluster-cookbook/pull/1648